bedrock/firefox/views.py

```diff
@@ -68,6 +69,9 @@ def installer_help(request):
     if installer_lang and installer_lang in firefox_desktop.languages:
         context['installer_lang'] = installer_lang
 
+    if installer_channel and installer_channel == 'aurora':
+        context['installer_channel'] = 'alpha'
+
     if installer_channel and installer_channel in INSTALLER_CHANNElS:
```

Comment:
The conditional you added above doesn't seem to be working, because it is overwritten by this conditional here below. Something like this should work?

```suggestion
    if installer_channel and installer_channel in INSTALLER_CHANNElS:
        if installer_channel == 'aurora':
            context['installer_channel'] = 'alpha'
        else:
            context['installer_channel'] = installer_channel
```

Reply:
ah whoops, thank you Alex.